packages/shared/providers.yaml

```diff
@@ -787,6 +787,27 @@ calendly:
         - owner
     docs: https://docs.nango.dev/integrations/all/calendly
 
+brightcrowd:
+    display_name: BrightCrowd
+    categories:
+        - alumni-data
+    auth_mode: BASIC
+    token_url: https://bcb-staging.auth.us-east-1.amazoncognito.com/oauth2/token
```

Comment:
Is staging correct? 
they don't advertise any API on their website, not confident about this URL